charts/pega/charts/installer/templates/_helpers.tpl

```diff
@@ -95,6 +95,14 @@
       value: "{{ template "k8sWaitForWaitTime" $ }}"
     - name: MAX_RETRIES
       value: "{{ template "k8sWaitForMaxRetries" $ }}"
+  resources:
+    # CPU and Memory that the init containers request
+    requests:
+      cpu: 50m
+      memory: 64Mi
+    limits:
+      cpu: 50m
+      memory: 64Mi
```

Comment:
Is any base reference or data available to conclude these values?

Reply:
Not exactly, so I set the limits to low values as it won't need much resources for the initContainers.